lighthouse-core/test/test-utils.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env jest */
+
+const i18n = require('../lib/i18n');
+
+expect.extend({
+  toHaveDisplayValue(received, expected) {
+    const obj = {displayValue: received.displayValue};
```

Comment:
it looks like this and logging is the only `displayValue` specific stuff in here. Do we want a more generalized `toHaveLocalizedWhatever()`?